kloppy/infra/serializers/tracking/epts/serializer.py

```diff
@@ -31,7 +31,7 @@ def __validate_inputs(inputs: Dict[str, Readable]):
     @staticmethod
     def _frame_from_row(row: dict, meta_data: EPTSMetaData) -> Frame:
         timestamp = row['timestamp']
-        if meta_data.periods:
+        if meta_data.periods and row['period_id'] != None:
```

Comment:
Compare with None should be `row['period_id'] is not None` or if period_id cannot be `0` it can be: `row['period_id']`